### PR TITLE
[Bug]ref bug #130582 #130641 #130626 - fix role of user and change name title

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**
[Bug] Unpaid leave of absence annotation is displayed with Emp code I/P
Count data: Title of count data items display incorrectly
Count data: Total of paid leave of absence, Total of unpaid leave of absence still display with Emp code I/P
Link: https://dev.framgia.com/redmine/issues/130582
https://dev.framgia.com/redmine/issues/130641
https://dev.framgia.com/redmine/issues/130626
Effect: N/A

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -249,11 +249,11 @@
     <string name="saturday_sunday">Saturday/sunday</string>
     <string name="your_notification">Your notification</string>
 
-    <string name="total_day_off">Total Day Off</string>
-    <string name="number_day_off_have_salary">Number Day Off Have Salary</string>
-    <string name="number_day_off_no_salary">Number Day Off No Salary</string>
-    <string name="number_over_time">Number Over Time</string>
-    <string name="total_fining">Total Fining</string>
+    <string name="total_day_off">Total of days off</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_day_off`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="number_day_off_have_salary">Total of paid leave of absence</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`number_day_off_have_salary`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="number_day_off_no_salary">Total of unpaid leave of absence</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`number_day_off_no_salary`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="number_over_time">Total of overtime</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`number_over_time`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="total_fining">Total of times for fining</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_fining`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java [hunk 6]
```diff
@@ -87,6 +94,18 @@ public void onGetTimeSheetSuccess(UserTimeSheet userTimeSheet) {
         setLoading(false);
     }
 
+    @Override
+    public void onGetUserSuccess(User user) {
+        mUser = user;
+        mIsFullTime = STAFF_FULL_TIME.equals(mUser.getNameStaffType());
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java]: pls move ``STAFF_FULL_TIME.equals(mUser.getNameStaffType());`` to ``User`` class
you should write method ``isFulltime()`` return boolean inside ``User`` class

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: pls change key of string to matching with values, And you must edit values for vietnamese

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: like above

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: pls change key of string to matching with values, And you must edit values for vietnamese

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -249,11 +249,11 @@
     <string name="saturday_sunday">Saturday/sunday</string>
     <string name="your_notification">Your notification</string>
 
-    <string name="total_day_off">Total Day Off</string>
-    <string name="number_day_off_have_salary">Number Day Off Have Salary</string>
-    <string name="number_day_off_no_salary">Number Day Off No Salary</string>
-    <string name="number_over_time">Number Over Time</string>
-    <string name="total_fining">Total Fining</string>
+    <string name="total_of_days_off">Total of days off</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_of_days_off`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="total_of_paid_leave_of_absence">Total of paid leave of absence</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_of_paid_leave_of_absence`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="total_of_unpaid_leave_of_absence">Total of unpaid leave of absence</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_of_unpaid_leave_of_absence`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="total_of_overtime">Total of overtime</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_of_overtime`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -249,11 +249,11 @@
...
+    <string name="total_of_times_for_fining">Total of times for fining</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`total_of_times_for_fining`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/java/com/framgia/wsm/data/model/User.java [hunk 12]
```diff
@@ -307,4 +309,8 @@ public boolean isManage() {
     public void setManage(boolean manage) {
         mIsManage = manage;
     }
+
+    public boolean isFullTime() {
+        return mNameStaffType.equals(STAFF_FULL_TIME);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/data/model/User.java]: pls change to 
```java
return StringUtils.isNotBlank(mNameStaffType) && mNameStaffType.equals(STAFF_FULL_TIME);
```

File: app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java [hunk 13]
```diff
@@ -45,6 +49,8 @@
     private boolean isVisibleFloatingActionMenu;
     private boolean mIsLoading;
     private boolean mIsRefreshEnable;
+    private User mUser;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java]: pls remove this variable

File: app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java [hunk 14]
```diff
@@ -87,6 +94,18 @@ public void onGetTimeSheetSuccess(UserTimeSheet userTimeSheet) {
         setLoading(false);
     }
 
+    @Override
+    public void onGetUserSuccess(User user) {
+        mUser = user;
+        mIsFullTime = mUser.isFullTime();
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java]: pls change to 
```java
mIsFullTime = user.isFullTime();
```

File: app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java [hunk 15]
```diff
@@ -45,6 +49,8 @@
     private boolean isVisibleFloatingActionMenu;
     private boolean mIsLoading;
     private boolean mIsRefreshEnable;
+    private User mUser;
+    private boolean mIsFullTime;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java]: pls rename this variable to ``mIsEmployeeFullTime``

File: app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java [hunk 16]
```diff
@@ -31,6 +33,8 @@
  */
 
 public class TimeSheetViewModel extends BaseObservable implements TimeSheetContract.ViewModel {
+    private static final String TAG = "TimeSheetViewModel";
+    public static final String STAFF_FULL_TIME = "Full-time";
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/timesheet/TimeSheetViewModel.java]: pls move this line to ``User`` class

**Pull request merged**